scripts/generate_crds_manifests.sh

```diff
@@ -36,24 +39,39 @@ done
 
 sed -i "s/^[Vv]ersion:.*\$/version: ${ver}/" "${chartdir}/Chart.yaml"
 
-go run helm.sh/helm/v3/cmd/helm template -n olm -f "${tmpdir}/values.yaml" --include-crds --output-dir "${chartdir}" "${chartdir}"
-
+${HELM} template -n olm -f "${tmpdir}/values.yaml" --include-crds --output-dir "${chartdir}" "${chartdir}"
 cp -R "${chartdir}"/olm/{templates,crds}/. "./manifests"
 
-for f in ./manifests/*.yaml; do
-   if [[ ! "$(basename "${f}")" =~ .*\.deployment\..* ]]; then
-      ${YQ} w -d'*' --inplace --style=double $f 'metadata.annotations['include.release.openshift.io/ibm-cloud-managed']' true
-   else
-      g="${f/%.yaml/.ibm-cloud-managed.yaml}"
-      cp "${f}" "${g}"
-      ${YQ} w -d'*' --inplace --style=double $g 'metadata.annotations['include.release.openshift.io/ibm-cloud-managed']' true
-      ${YQ} d -d'*' --inplace $g 'spec.template.spec.nodeSelector."node-role.kubernetes.io/master"'
-   fi
-   ${YQ} w -d'*' --inplace --style=double $f 'metadata.annotations['include.release.openshift.io/self-managed-high-availability']' true
-   ${YQ} w -d'*' --inplace --style=double $f 'metadata.annotations['include.release.openshift.io/single-node-developer']' true
-done
+add_ibm_managed_cloud_annotations() {
+   local manifests_dir=$1
 
-# requires gnu sed if on mac
-find ./manifests -type f -exec sed -i "/^#/d" {} \;
-find ./manifests -type f -exec sed -i "1{/---/d}" {} \;
+   for f in "${manifests_dir}"/*.yaml; do
+      if [[ ! "$(basename "${f}")" =~ .*\.deployment\..* ]]; then
+         ${YQ} w -d'*' --inplace --style=double "$f" 'metadata.annotations['include.release.openshift.io/ibm-cloud-managed']' true
+      else
+         g="${f/%.yaml/.ibm-cloud-managed.yaml}"
+         cp "${f}" "${g}"
+         ${YQ} w -d'*' --inplace --style=double "$g" 'metadata.annotations['include.release.openshift.io/ibm-cloud-managed']' true
+         ${YQ} d -d'*' --inplace "$g" 'spec.template.spec.nodeSelector."node-role.kubernetes.io/master"'
+      fi
+      ${YQ} w -d'*' --inplace --style=double "$f" 'metadata.annotations['include.release.openshift.io/self-managed-high-availability']' true
+      ${YQ} w -d'*' --inplace --style=double "$f" 'metadata.annotations['include.release.openshift.io/single-node-developer']' true
+   done
+}
+
+update_csv() {
+   local csv=$1
+
+   yq w --inplace "${csv}" --tag '!!bool' 'spec.cleanup.enabled' false
+   yq w --inplace "${csv}" 'spec.customresourcedefinitions' {}
+   yq w --inplace "${csv}" --style="" 'spec.install.spec.deployments[0].spec.template.spec.containers[0].ports[0].protocol' TCP
+   yq w --inplace "${csv}" --style="" 'spec.install.spec.deployments[0].spec.template.metadata.creationTimestamp' null
+   sed -i "s/'{}'/{}/g" "${csv}"
```

Comment:
This is needed as the `yq w --inplace "${csv}" 'spec.customresourcedefinitions' {}` command above outputs a `'{}'` as the default style. I wasn't able to find a way to do this natively using yq v3, as there weren't any non-default styles that were removing the quotes as needed, or specifying `--tag '!!map'` was being recognized as a custom tag, vs. explicitly casting to a map key. This should be fine in the short-term as this resource will be removed from CVO control (and thus from this root list of manifests) in 4.9+.